### ISSUE TITLE
Fish/shell autocompletion, tldr and manpages

### ISSUE THREAD

**Issue by abraham-sah** [Slackadays/Clipboard#138]
It's three different things so may you want to remove this issue and turn it to three ones...

I installed it and am using it in Arch through AUR bin package from long time ago.

Your project is a work of art. Many advantages over xsel and xclip.

But honestly! No man-pages? No tldr? No shell autocompletion?? How a remote admin not having access to a browser is supposed to quickly find his desired subcommand/flag for his specific task? Opening a browser in his smartphone or some other device?

Perhaps it's provided already somewhere but I can't find it?

Thanks in advance...

**Comment by Slackadays** [comment]
I'd love to have all the info in a nice little manpage, but the problem last time I tried that was synchronizing all the info between the GitHub readme and the getclipboard.app website and the manpage. Is there a better solution you have in mind?

**Comment by Slackadays** [comment]
@abraham-sah added bash and fish completions in https://github.com/Slackadays/Clipboard/commit/5f30d7a2d13af001fc8a4865caaa405d5514cff3

**Comment by Slackadays** [comment]
tldr now in the staging phase at https://github.com/tldr-pages/tldr/pull/10414

**Comment by Slackadays** [comment]
Merged

**Comment by Slackadays** [comment]
Almost ready to spruct up the existing (but barren) manpage

**Comment by Slackadays** [comment]
@abraham-sah just made the manpages not suck in https://github.com/Slackadays/Clipboard/commit/01f764b991912990e8d2cc1c474f72db932c6731, is there anything else you'd like?

**Comment by abraham-sah** [comment]
> @abraham-sah just made the manpages not suck in [01f764b](https://github.com/Slackadays/Clipboard/commit/01f764b991912990e8d2cc1c474f72db932c6731), is there anything else you'd like?
> 
> I can reopen the issue if you need more :)

Going to check them out on my next upgrade. But they already look very well in here in commits.

Thanks a lot.

**Comment by reagle** [comment]
Hi, just found this, I always wanted a file clipboard for the terminal, thanks!  However, I can't find an zsh completions and I notice this issue is closed. Are there any?

**Comment by Slackadays** [comment]
@reagle There should be some here: https://github.com/Slackadays/Clipboard/blob/main/documentation/completions/cb.zsh

How did you install CB?

**Comment by reagle** [comment]
Hi. I installed via homebrew. They are available in that package: `/opt/homebrew/opt/clipboard/share/zsh/site-functions/_cb`. And I access them if I add that to the fpath manually: `fpath=(/opt/homebrew/opt/clipboard/share/zsh/site-functions $fpath)` but I'm not sure why I have to do that. I've never done that before (I'm no zsh expert).
